src/compute/src/render/threshold.rs

```diff
@@ -91,9 +92,10 @@ where
     L: Fn(&Diff) -> bool + 'static,
 {
     match oks {
-        SpecializedArrangementImport::Bytes9Row(key_types, inner) => {
-            let oks = threshold_arrangement(inner, name, logic);
-            SpecializedArrangement::Bytes9Row(key_types.clone(), oks)
+        SpecializedArrangementImport::RowUnit(inner) => {
+            let name = format!("{} [val: empty]", name);
+            let oks = threshold_arrangement(inner, name.as_str(), logic);
+            SpecializedArrangement::RowUnit(oks)
```

Comment:
Uncovered, interesting

Reply:
Can we know here if the other arm of the `match` statement is covered by our tests on main? It seems to be hard to come up with a SQL statement where we can reuse a trace here (e.g., originating from an index), because threshold is always planned on top of a union when doing `EXCEPT` or `EXCEPT ALL`.

Reply:
Sure, picking another coverage run (https://buildkite.com/materialize/coverage/builds/271#018b44b7-353b-484f-9358-3382537b14cf) I downloaded the full coverage report and the entire function is uncovered:
![Screenshot 2023-10-19 at 14 21 52](https://github.com/MaterializeInc/materialize/assets/2335377/e52a4fe7-c559-4717-8269-67a79f7c5ccb)

Reply:
To be more specific, what I am looking for is if any of our current tests cover the following lines: https://github.com/MaterializeInc/materialize/blob/ecd7c56ade393b0d0479e403dadafd9008c8eff7/src/compute/src/render/threshold.rs#L128-L129

Reply:
> Sure, picking another coverage run (https://buildkite.com/materialize/coverage/builds/271#018b44b7-353b-484f-9358-3382537b14cf) I downloaded the full coverage report and the entire function is uncovered:

Oh, cool, I had not seen your comment before! So this is as I expected. I don't think that it is possible to trigger this code path from SQL the way we do query planning right now. It would be the kind of thing that we could test one day as part of a SQL-agnostic Compute layer interface, e.g., when we address #11097 or #13769 in the future.